distribution/examples/security/cors/membrane.sh

```
@@ -0,0 +1,73 @@
+}
+
+start_membrane() {
+    membrane_home=$(find_membrane_directory "$(pwd)")
+    if [ $? -eq 0 ]; then
+        start "$membrane_home"
+    else
+        echo "Could not start Membrane. Ensure the directory structure is correct."
+    fi
+}
+
+if ! ( _test=test && _="${_test#t}" ) >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[warning] Use var=$(command) to assign output (or quote to assign string).

(SC2209)